interfaces/builtin/lxd_support_test.go

```diff
@@ -95,7 +113,7 @@ func (s *LxdSupportInterfaceSuite) TestAppArmorSpecUserNS(c *C) {
 func (s *LxdSupportInterfaceSuite) TestAppArmorSpecUnconfined(c *C) {
 	spec := &apparmor.Specification{}
 	c.Assert(spec.AddPermanentPlug(s.iface, s.plugInfo), IsNil)
-	c.Assert(spec.Unconfined(), Equals, true)
+	c.Assert(spec.Unconfined(), Equals, apparmor.UnconfinedSupported)
```

Comment:
I was expecting at least a test that shows the effective value is `UnconfinedEnabled` (not in this test).

Reply:
Good point - added in 8481462104